tests/gui/pages/main.spec.js

```diff
@@ -338,7 +338,7 @@ describe('Browser: MainPage', function() {
           });
 
           settings.set('unmountOnSuccess', false);
-          m.chai.expect(controller.getProgressButtonLabel()).to.equal('50%');
+          m.chai.expect(controller.getProgressButtonLabel()).to.equal('50% Flashing');
```

Comment:
Can we make all the tests that test `getProgressButtonLabel` test `makeSubtitle` instead? Once that's done, the ``getProgressButtonLabel` can be greatly reduced to only test the "Flash" edge case.